src/Asset.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+/*
+ * This file is part of the limit0/assets package.
+ *
+ * (c) Limit Zero, LLC <contact@limit0.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Limit0\Assets;
+
+use Symfony\Component\HttpFoundation\File\UploadedFile;
```

Comment:
@zarathustra323 Looks like it needs to stay for now. Discovered that it wasn't actually storing the ContentType with the S3 call, which is why I initially thought we could remove the dep.

Specifically it just needs the MimeTypeGuesser, but that doesn't appear to be available outside of symfony/http-foundation,